src/haggadah/views.cljs

```diff
@@ -122,40 +122,101 @@
+        [:div.control 
+         [:a.button.is-small {:class (styles/submit-button) :on-click  #(re-frame/dispatch [::events/create-seder id
+                                                                                            (form-content "seder-title")])
+                              :data-testid :submit} "Create"]]]]]
+     [:button.modal-close.is-large]]))
+
+
 (defn dashboard-panel
   []
   [:div.page 
    [:div.container.is-large.hero.is-flex
     [:div.hero-body.pt-6
-     [:div.pt-24.column
-      (let [name (re-frame/subscribe [::subs/name])]
+     (let [haggadot @(re-frame/subscribe [::subs/haggadot])
+           sedarim @(re-frame/subscribe [::subs/sedarim])]
+       [:div.pt-24.column
+        (let [name (re-frame/subscribe [::subs/name])]
+          [:div
+           [:h1.text-center.is-size-4 {:data-testid :user}
+            (str "Hello " @name ". Welcome. To make a new Haggadah, click the button to your right. To share and edit your existing Haggadah, look at your Haggadot below ")]])
+        [:div.pl-6.buttons.is-right
+         [:a.button.is-smalll.is-pulled-right.mt-2 {:data-testid :create-haggadah
+                                                    :on-click #(re-frame/dispatch [::push-state :haggadah-creation])}   "Create Haggadah"]]
         [:div
-         [:h1.text-center.is-size-4 {:data-testid :user}
-          (str "Hello " @name ". Welcome. To make a new Haggadah, click the button to your right. To share and edit your existing Haggadah, look at your Haggadot below ")]])
-      [:div.pl-6.buttons.is-right
-       [:a.button.is-smalll.is-pulled-right.mt-2 {:data-testid :create-haggadah
-                                                        :on-click #(re-frame/dispatch [::push-state :haggadah-creation])}   "Create Haggadah"]]
-      [:div
-       [:h1.is-size-3
-        "Haggadot created"]
-       (let [haggadot @(re-frame/subscribe [::subs/haggadot])]
+         [seder-popup]
+         [:h1.is-size-3
+          "Haggadot created"]
          (when haggadot
            [:ul.haggadot 
             (for [{:keys [title id]} haggadot :when id] 
               ^{:key id}[:li.mb-2
-                         [:a.haggadah-link {:data-testid :haggadah-link 
-                              :href (href :haggadah-view {:id id})} title]])]))] ]]]
-   [wave-bottom]])
+                         [:a.haggadah-link.mr-2 {:data-testid :haggadah-link 
+                                                 :href (href :haggadah-view {:id id})} title]
+                         [:a.button.is-small {:data-testid :create-seder
+                                              :on-click #(re-frame/dispatch [::events/create-seder-modal id])} "Create Seder"]])])]
 
+        [:h1.is-size-3.pt-3
+         "Sedarim"]
+        (when sedarim
+          [:ul.sedarim 
+           (for [{:keys [title id]} sedarim :when id] 
+             ^{:key id}[:li.mb-2
```

Comment:
NIT: From the screenshots it would be look nicer to align the haggadah name to certain width and the button to the right... like a table....

Reply:
I'll make an issue which will deal with aligning the haggadah name and button.